src/utils/misc.ts

```diff
@@ -133,15 +138,24 @@ export const encrypt: (
   encryptionKey: string,
 ) => Promise<{ iv: string; content: string } | undefined> = async (text, encryptionKey) => {
```

Comment:
change this type to Promise<...> without `|undefined` to avoid the `?` functions should only return what is expected if they throw an error no need to return `undefined`